src/apply-configuration.coffee

```diff
@@ -91,8 +91,8 @@ applyConfiguration = (config) ->
   if config.blueprintPath
     configuration.options.path.push config.blueprintPath
 
-  configuration.options.method.map((method) ->
-    method.toUpperCase()
+  configuration.options.method = (
```

Comment:
The thing is, the Dredd option is `--method`, but accepts also multiple values, so in configuration handling, it's coerced to array while the original singular name sticks. Not ideal and it's on multiple places in Dredd currently. Confused me too, at the first sight.